Headers/Public/VLCMediaPlayer.h

```diff
@@ -875,6 +889,28 @@ extern NSString *const VLCTitleDescriptionIsMenu;
 @property (NS_NONATOMIC_IOSONLY, readonly) NSImage *lastSnapshot;
 #endif
 
+/**
+ * Start recording at given **directory** path
+ * \param path directory where the recording should go
```

Comment:
okay just to clarify, you don't care about the local file part but about it being an NSURL instead of an NSString, correct? in that case I'm happy to change it to startRecordingAtDirectoryURL:(NSURL *)directoryURL